crypto/x509/v3_addr.c

```diff
@@ -1272,7 +1271,7 @@ static int addr_validate_path_internal(X509_STORE_CTX *ctx,
                 IPAddressFamily *fc = sk_IPAddressFamily_value(child, j);
 
                 if (!IPAddressFamily_check_len(fc))
-                    return 0;
+                    goto done;
```

Comment:
Any returns after line 1254 should be goto's

Reply:
Didn't realise this

Reply:
Quite a common problem we get in code..